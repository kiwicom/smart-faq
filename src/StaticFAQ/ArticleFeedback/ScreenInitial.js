// @flow

import * as React from 'react';
import { Text, Button } from '@kiwicom/orbit-components';

import screensList from './screensList';
import Box from '../../common/Box';

type Props = {|
  changeScreen: (nextScreen: string) => void,
|};

const ScreenInitial = (props: Props) => (
  <Box
    border="none"
    padding="16px 24px"
    borderRadius="4px"
    backgroundColor="#f5f7f9"
    mobileBackgroundColor="transparent"
  >
    <div className="question">
      <Text type="secondary">
        Didn&apos;t find the answer you were looking for?
      </Text>
    </div>
    <div className="button">
      <Button
        type="secondary"
        title="Let us know"
        width={115}
        onClick={() => props.changeScreen(screensList.INPUT)}
      />
    </div>
    <style jsx>
      {`
        div.question {
          display: flex;
          flex-direction: column;
          align-items: center;
          margin-bottom: 16px;
        }
        .button {
          display: flex;
          justify-content: center;
        }
      `}
    </style>
  </Box>
);

export default ScreenInitial;
