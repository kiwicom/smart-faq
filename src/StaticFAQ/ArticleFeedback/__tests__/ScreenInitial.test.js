// @flow

import * as React from 'react';
import { shallow } from 'enzyme';

import ScreenInitial from '../ScreenInitial';

const ch = (sc: string) => {}; //eslint-disable-line

describe('SignIn', () => {
  const component = shallow(<ScreenInitial changeScreen={ch} />);
  it('should match snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
