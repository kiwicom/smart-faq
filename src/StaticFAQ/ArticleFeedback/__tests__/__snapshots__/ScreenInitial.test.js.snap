// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScreenInitial should match snapshot 1`] = `
<Box
  backgroundColor="#f5f7f9"
  border="none"
  borderRadius="4px"
  mobileBackgroundColor="transparent"
  padding="16px 24px"
>
  <div
    className="jsx-3628750341 question"
  >
    <Text
      type="secondary"
    >
      Didn't find the answer you were looking for?
    </Text>
  </div>
  <div
    className="jsx-3628750341 button"
  >
    <WithTheme(Button)
      onClick={[Function]}
      type="secondary"
      width={115}
    >
      Let us know
    </WithTheme(Button)>
  </div>
  <JSXStyle
    css="div.question.jsx-3628750341{display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-flex-direction:column;-ms-flex-direction:column;flex-direction:column;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;margin-bottom:16px;}.button.jsx-3628750341{display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-box-pack:center;-webkit-justify-content:center;-ms-flex-pack:center;justify-content:center;}"
    styleId="3628750341"
  />
</Box>
`;
